src/SLCore/Configuration/ISLCoreConstantsProvider.cs

```diff
@@ -27,4 +28,8 @@ public interface ISLCoreConstantsProvider
     ClientConstantsDto ClientConstants { get; }
     FeatureFlagsDto FeatureFlags { get; }
     TelemetryClientConstantAttributesDto TelemetryConstants { get; }
+    
+    IReadOnlyList<Language> LanguagesInStandaloneMode { get; }
+    IReadOnlyList<Language> ExtraLanguagesInConnectedMode { get; }
+    IReadOnlyList<Language> AnalyzableLanguages { get; }
```

Comment:
Remove the connected mode languages (for now).
Rename the AnalyzableLanguages to reflect they are analyzed by sloop.